### PR TITLE
Add parser for the satellite clients count spec

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### CODE REVIEW

**Comment by huali027** [comment]
The spec is approved at https://gitlab.cee.redhat.com/insights-release-eng/insights-core-assets/-/merge_requests/128

File: insights/parsers/satellite_content_hosts_count.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+"""
+SatelliteContentHostsCount - command ``/usr/bin/su - postgres -c "psql -d foreman -c 'select count(*) from hosts'"``
+====================================================================================================================
```
Comment (xiangce) [inline insights/parsers/satellite_content_hosts_count.py:3]: The command can be short as `psql -d foreman -c 'select count(*) from hosts'`

File: insights/specs/default.py [hunk 2]
```diff
@@ -606,6 +606,7 @@ def sap_sid(broker):
     sap_hdb_version = foreach_execute(sap_sid, "/usr/bin/sudo -iu %sadm HDB version", keep_rc=True)
     saphostctl_getcimobject_sapinstance = simple_command("/usr/sap/hostctrl/exe/saphostctrl -function GetCIMObject -enuminstances SAPInstance")
     sat5_insights_properties = simple_file("/etc/redhat-access/redhat-access-insights.properties")
+    satellite_content_hosts_count = simple_command("/usr/bin/su - postgres -c \"psql -d foreman -c 'select count(*) from hosts'\"")
```
Comment (bfahr) [inline insights/specs/default.py:609]: @huali027 this spec should use the `sudo` command instead:
```
"/usr/bin/sudo -iu postgres psql -d foreman -c 'select count(*) from hosts'"
```

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/specs/default.py (hunk 2 above)
Comment (huali027) [reply insights/specs/default.py:609]: @bfahr, sure, I'll update it, thanks for your review.

**Review by xiangce — APPROVED** [review]

**Pull request merged**